src/SFA.DAS.LevyTransferMatching/Application/Queries/Pledges/GetApplications/GetApplicationsQueryHandler.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using SFA.DAS.LevyTransferMatching.InnerApi.LevyTransferMatching.Requests;
+using SFA.DAS.LevyTransferMatching.Interfaces;
+
+namespace SFA.DAS.LevyTransferMatching.Application.Queries.Pledges.GetApplications
+{
+    public class GetApplicationsQueryHandler : IRequestHandler<GetApplicationsQuery, GetApplicationsQueryResult>
+    {
+        private readonly ILevyTransferMatchingService _levyTransferMatchingService;
+
+        public GetApplicationsQueryHandler(ILevyTransferMatchingService levyTransferMatchingService)
+        {
+            _levyTransferMatchingService = levyTransferMatchingService;
+        }
+
+        public async Task<GetApplicationsQueryResult> Handle(GetApplicationsQuery request, CancellationToken cancellationToken)
+        {
```

Comment:
As per previous comment, we could go get the standards here as well (and await them both together) and then assemble a single response.

Reply:
Agreed, I'll implement this